unity/Holo/Assets/Editor/ModelLoad/ModelImport/VTKImport/PolyDataImporter.cs

```diff
@@ -121,10 +121,11 @@ private bool GetSimulationColors(StreamReader streamReader)
         {
             DeltaTangents = new Vector3[Vertices.Length];
             streamReader.ReadLine();
-            for (int i = 0; i < DeltaTangents.Length; i++)
+            for (int i = 0; i < DeltaTangents.Length; i+=2)

```

Comment:
Because in each flow line there are two vertices and they are of the same colour (in VTK files we got, colour is saved for a cell instead of a vertex, so that's how I act here.). However this code will be deprecated after this PR, in favour of our external app.